list_of_subprojects.md

```diff
@@ -0,0 +1,71 @@
+# List of Official Jupyter Subprojects
+
+At the highest level, any Github repository under any official Jupyter GitHub organization is either itself an Official Jupyter Subproject, or is part of an Official Jupyter Subproject. All such Subprojects have all of the privileges and responsibilities detailed in the Software Subprojects of Jupyter governance document. Within the Official Jupyter Subproject designation, there are two types of Subprojects: ones with a formal decision making body and SSC representation, and smaller, less active ones whose decision making body is the SSC itself. This document enumerates Subprojects of these two types.
+
+### Official Subprojects with SSC representation
+
+The following Jupyter Subprojects have their own formal decision making body that elects and maintains an SSC representative:
+
+- JupyterLab (https://github.com/jupyterlab/jupyterlab)
+- JupyterHub and Binder 
+  - https://github.com/jupyterhub/jupyterhub
+  - https://github.com/jupyterhub/binder
+  - https://github.com/jupyterhub/binderhub
+- Voilà (https://github.com/voila-dashboards/voila)
+- Jupyter Server (https://github.com/jupyter-server/jupyter_server)
+- Jupyter Widgets (https://github.com/jupyter-widgets)
+- Jupyter Notebook (https://github.com/jupyter/notebook)
+- Jupyter Kernels
+  - jupyter-xeus (https://github.com/jupyter-xeus/)
+  - IPykernel (https://github.com/ipython/ipykernel)
+  - IPython (https://github.com/ipython/ipython)
+- Jupyter Foundations
+- Jupyter Standards
+- Jupyter UX, Design and Branding
+- Jupyter Enterprise
+
+### Official Subprojects without SSC representation
+
+The following official Jupyter Subprojects are smaller and less active. As such their formal decision making body will be the SSC and they will not have an independent SSC delegate. Our expectation is that these smaller teams will basically continue working as they do today, making decisions using the consensus seeking phase of the Jupyter Decision Making Guidelines. In the rare situation that a vote is called, the SSC will serve as the voting body and will consult closely with the team. If one of these Subprojects grows or becomes more active, the SSC can, at any time, elect a standalone decision making body for the Subproject, at which point, the Subproject will begin to have an SSC delegate of their own. In all other respects, these Subprojects are full and official Subprojects.
```

Comment:
Should we explicitly state that even though the ssc has decision making authority over these projects, the ssc delegates all decisions that aren’t `some major criteria` to whoever has commit rights in those repositories?

Reply:
Good point, thx! Changes made based on your feedback/language.